### ISSUE TITLE
hist is exported, but undefined

### ISSUE THREAD

**Issue by tpapp** [JuliaStats/StatsBase.jl#338]
The symbol `hist` is [exported](https://github.com/JuliaStats/StatsBase.jl/blob/e75d9ad28f29da868171f80e56291708e2e2bdf5/src/StatsBase.jl#L131), but appears to have no definition. Possibly the remnant of an earlier interface?

**Comment by ararslan** [comment]
Yes, I believe so. Thanks, good catch.

**Comment by andreasnoack** [comment]
There has been some discussion about reintroducting `hist` since `fit(Histogram, x)` is a bit verbose. I'd actually prefer `histogram` though.

**Comment by tpapp** [comment]
FWIW, I like `fit(Histogram, ...)` a lot. I was thinking about extending this to other non-parametric summaries, such as `fit(ECDF, ...)`.

In any case, I think that exporting `hist` is currently a bug, and should be removed, no matter what future plans are. When those materialize, it can be reintroduced. I opened this issue following [this dicussion](https://discourse.julialang.org/t/function-hist-not-found/8290), where a user was confused, possibly following some earlier tutorial.

**Comment by andreasnoack** [comment]
I agree that it is a bug and it is easy to reintroduce later if we change our minds. It just reminded me about the discussion of `hist` but let's take that discussion elsewhere.

**Comment by nilshg** [comment]
I believe `hist` isn't exported any longer so this can be closed?
